src/Microsoft.Diagnostics.Monitoring.WebApi/Utilities/StackUtilities.cs

```diff
@@ -55,7 +55,12 @@ internal static Models.CallStackFrame CreateFrameModel(CallStackFrame frame, Nam
             else if (cache.FunctionData.TryGetValue(frame.FunctionId, out FunctionData functionData))
             {
                 frameModel.ModuleName = NameFormatter.GetModuleName(cache, functionData.ModuleId);
-                frameModel.MethodName = functionData.Name;
+
+                builder.Clear();
+                builder.Append(functionData.Name);
+
+                NameFormatter.BuildMethodParameterTypes(builder, cache, functionData.ParameterTypes);
```

Comment:
@wiktork, do you think this is okay (the method name includes the name and the parameter types) or should the parameter types be another property on the CallStackFrame?

Reply:
@kkeirstead, what if you add a parameter to `CreateFrameModel` named `bool excludeMethodParameters = false`, skip the `buildMethodParameterTypes` call when this is true, and then set this parameter to true in the stacks feature usage? That way the existing behavior for stacks is preserved and can be later flipped back to false if the feature starts supporting method parameters.

Reply:
Can we keep the structured data separate and then use the param types in the method name at rendering time?

Reply:
> Can we keep the structured data separate and then use the param types in the method name at rendering time?

That makes sense.

Reply:
I think my latest change addresses this point, unless I misinterpreted what you had in mind.